tests/data/tiny/README.md

```diff
@@ -0,0 +1,8 @@
+# Test data set provenance
+
+## Authors
+
+- Guilherme Dias
+- Martin Pippel
+
+## Provenance
```

Comment:
```suggestion
## Provenance

- species: _Drosophila melanogaster_ 
- PacBio HiFi data was downloaded from NBCI´s SRA [SRR10238607](https://www.ncbi.nlm.nih.gov/sra/SRR10238607)
- Illumina HiC data was downloaded from NBCI´s SRA [SRR10512944](https://www.ncbi.nlm.nih.gov/sra/SRR10512944)

- PacBio HiFi data were randomly subsampled to 15X read coverage and assembled with [hifiasm version 0.19.8](https://github.com/chhylp123/hifiasm)
- PacBio reads were mapped back to the genome assembly with minimap2
- Illumina HiC reads were mapped back to the genome assembly with bwa mem

- PacBio and HiC reads were extracted from the alignment bam files: 
   - from a 2Mb region within a 28Mb contig
   - a 20Kb gap was introduces at Position 1Mb and all PacBio and HiC reads that mapped into that region were filtered out
   - HiC reads were further filtered down to 50%, representing a 38X coverage 
```

Reply:
Could you add code here too please? Did you use `wget` to fetch the reads.